### ISSUE TITLE
Add pixie-operator pod logs to `px collect-logs` CLI output. 

### ISSUE THREAD

**Issue by htroisi** [pixie-io/pixie#559]
Pixie's CLI allows you to gather Pixie logs using the `px collect-logs` command. The output logs should also include the pixie-operator pod logs, if applicable.

**Comment by victor-timofei** [comment]
Hi, I saw this marked as a good first issue. Could I try picking it up?

**Comment by htroisi** [comment]
@victor-timofei - Yes, we'd be happy have you contribute this fix! 

I have a suspicion that this has to do with the fact that the `pixie-operator` pod does not have an `app` label, like the rest of the Pixie pods:
<img width="2220" alt="image" src="https://user-images.githubusercontent.com/66266496/184994160-3ef65809-70d4-4065-84d4-e1b2680f9ad5.png">

To test out your changes to the Pixie CLI: 
```
cd pixie/src/pixie_cli
go run px.go <CLI_COMMAND>
# To test your changes to the `px collect-logs` command, you'd run `go run px.go collect-logs`
```

**Comment by victor-timofei** [comment]
Yeah, you are right. It is because the `app` label is missing.

I've noticed the `pixie-operator-index` pod is deployed by the OLM.
I did a fast check on the `CatalogSource` CRD to find what options it supports, and I could not find any way to add labels.

1. I could follow up with the OLM folks and see if there is any way to add labels to an operator(or if it would be possible to extend the CRD?).
2. I guess the operator could append the labels on itself(although I think this is a little hacky).
